crowbar_framework/lib/crowbar/error/upgrade_status.rb

```diff
@@ -24,7 +24,9 @@ def initialize(step_name = "")
 
     class StartStepRunningError < StandardError
       def initialize(step_name = "")
-        super("The step #{step_name} has already been started")
+        msg = "Some step is already running."
```

Comment:
Not sure what is the recommended way of doing such things in ruby but I'd switch to a regular `if ... else ...` to make this part clear.